tutorials/R/api-token/api-token.Rmd

```diff
@@ -0,0 +1,270 @@
+---
+syncID: 99d22b526c1a4a88847259a4d05b46af
+title: "Using an API Token when Accessing NEON Data"
+description: "Get an API token tied to your NEON user account, and use it for faster download speeds when accessing NEON data via the neonUtilities package."
+dateCreated: 2020-05-04
+authors: [Claire K. Lunch]
+contributors:
+estimatedTime: 20 minutes
+packagesLibraries: neonUtilities
+topics: data-management, rep-sci
+languageTool: R
+code1: R/api-token/api-token.R
+tutorialSeries:
+urlTitle: api-token
```

Comment:
Hmm - I just had another thought.  Joel with Aten wanted to make sure our URL names were unique enough to not possibly overlap with other webpages.  I can see this one being a possible webpage title too.  Suggest changing to something a bit longer/more specific or adding "tutorial".  perhaps neon-api-tokens-code-tutorial with corresponding file change?

Reply:
Works for me, just pushed the change!